plugin/adsk/scripts/AETemplateHelpers.py

```diff
@@ -0,0 +1,28 @@
+import ufe
+import mayaUsd.ufe
+
+def GetDefaultPrimName(proxyShape):
+    if not proxyShape.startswith('|world'):
+        proxyShape = '|world' + proxyShape
```

Comment:
We should not do this.  We should fix the implementation of getStage() in wrapUtils.cpp so that it converts the argument path string to a path using Ufe::PathString::path().

Reply:
As we discussed I cleaned up all the usage of "|world"